processor/probabilisticsamplerprocessor/testdata/invalid_inf.yaml

```diff
@@ -4,7 +4,7 @@ receivers:
 processors:
 
   probabilistic_sampler/traces:
-    sampling_percentage: +Inf
+    sampling_percentage: 999_999_999_999_999_999_999_999_999_999_999_999_999
```

Comment:
is `+Inf` no longer supported?

Reply:
I wasn't able to find a way to get it interpreted as a float. I tried `!!float +Inf` and got this error message:

```
cannot resolve the configuration: cannot retrieve the configuration: yaml: cannot decode !!str `+Inf` as a !!float"
```

Reply:
From https://yaml.org/spec/1.2.2/#1032-tag-resolution, `+.Inf` should work

Reply:
Updated this on dde0255f442ae162aa08ef58598f035f79089807

Reply:
Thanks!